src/interfaces/ISablierMerkleFactory.sol

```diff
@@ -75,6 +72,10 @@ interface ISablierMerkleFactory is IAdminable {
                                  CONSTANT FUNCTIONS
     //////////////////////////////////////////////////////////////////////////*/
 
+    /// @notice Retrieves the fee for the provided campaign creator, using the default fee if no custom fee is set.
+    /// @param campaignCreator The address of the user for whom the fee is being computed.
+    function computeFee(address campaignCreator) external view returns (uint256);
+
     /// @notice Retrieves the custom fee struct for the provided campaign creator.
     /// @param campaignCreator The address of the user for whom the details are being retrieved.
     function customFee(address campaignCreator) external view returns (MerkleFactory.CustomFee memory);
```

Comment:
And while we are at renaming fee-related functionality, could we also rename this getter to `getCustomFee`?

Reply:
OK.